src/utils/formatError.test.ts

```diff
@@ -33,7 +33,7 @@ describe('formatError', () => {
     };
     const client = new ImmutableX(config);
     await expect(client.getUser('')).rejects.toThrowError(
-      'Error: Request failed with status code 404',
+      'no matching operation was found',
```

Comment:
This test was failing on main too, something about our 404 response must have changed as this test hits the sandbox api